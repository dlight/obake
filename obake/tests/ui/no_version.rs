#[obake::versioned]
struct Foo {}

#[obake::versioned]
enum Bar {}

fn main() {}
