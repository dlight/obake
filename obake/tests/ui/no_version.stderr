error: `#[obake::versioned]` items require at least one `#[obake(version(...))]` attribute
 --> $DIR/no_version.rs:2:1
  |
2 | struct Foo {}
  | ^^^^^^

error: `#[obake::versioned]` items require at least one `#[obake(version(...))]` attribute
 --> $DIR/no_version.rs:5:1
  |
5 | enum Bar {}
  | ^^^^
